content/en/guides/contributing/minutes/index.md

```diff
@@ -17,6 +17,120 @@ Attendees:
 
 ---
 
+## 2023-12-19
+
+Attendees: Ali Smith, Daniel Wagner-Hall, Mitchell Lloyd, Sally McGrath, Zsolt Sztupak
+
+### Sprint planning
+
+Sprint is basically "review all the things".
+* Ali will make sure Node gets merged.
+* Daniel will make sure JS3 gets merged.
+* Sally will port React
+
+### ❗Actions from last time
+
+- [x] Ali (Carry-over): to put together a proposed ordering of training videos to watch, and arrange a time for discussion.
+- [ ] Ali (Carry-over): Review JS1 & 2 content
+- [x] Ali: Update meeting reminder bot to happen on Tuesdays not Thursdays
+- [ ] Ali: Find/Make ticket about reworking React
+- [x] Daniel: Read up on Zsolt's discussion about backend hosting platforms and engage (and also see if he's interested in reworking the full-stack assessment to be more feature-based)
+- [x] Sally: Share application process details with MigraCode folks
+- [ ] Isar: Triage JS1+JS2 Iteration and Snagging issues - add any needed to the MigraCode milestone
+- [ ] Isar: Get all the relevant MigraCode people to fill in [this form](https://docs.google.com/forms/d/1sp2MTsU0eEAr4HCKXfDPxUXXZROWN8UyHIrDk7dRSJk/edit) to join CYF GitHub teams.
+- [ ] Sally: Provide a repro case for https://github.com/CodeYourFuture/curriculum/issues/222
+- [ ] Ali: Write up a ticket for Sally to document the architecture of the curriculum website
+
+### 📝 Agenda points
+
+#### Curriculum repo layout
+
+* Monorepos make it simpler to do things atomically (e.g. for wide-scale change, for unified previews, etc), and to not need to build tooling around stitching things together.
+  * Each time we need to build tooling to bridge gaps, either we need to spend time on that tooling, or we need to be sad because we have to live with the limitations of not having that tooling.
+* Our core model is composibility - separating things around can make it easier for people to focus on just their corner of the world.
+* Each place we have an extra repo adds complexity - we should be careful to justify why this it's worth adding this complexity for each exception we make.
+* Not super clear what value we get from seprate _repos_ over separate _directories_ (which may have their own permissions)?
+  * But Hugo's module system strongly assumes each module has _releases_ of some kind, and we may need to invent some cunning scheme to make this work.
+* Git submodules could potentially help, but also could add complexity.
+* Sally is going to try to build some kind of release system within Hugo and send it out for high-bar review, outside of the curriculum repo so we can experiment and iterate.
+* After prototyping, will need to have branding discussions about exactly where the curriculum lives, if it's hosting multiple organisations' curricula.
+
+#### CYF+
+
+* Refresher on CYF+:
+  * Partnership with Slack - they're paying us to run the course, and are paying the trainees for a year to take the course and then guaranteeing them employment for the rest of the year.
```

Comment:
```suggestion
  * Partnership with Slack - they're paying the trainees to take the course and on successful completion, a fixed term contract. This will be our second run. First run they hired both trainees in permanent roles -- this partnership is not a job guarantee.
```